tests/test_quick_start.py

```diff
@@ -368,30 +373,19 @@ def _deserialize(config, raw_bytes, examples):
 @pytest.mark.parametrize("model,conf_file,model_dir,mode", TEST_GRID, scope='class')
 class TestQuickStart(object):
     @staticmethod
-    def interact(config_path, model_directory, qr_list=None):
-        qr_list = qr_list or []
-        logfile = io.BytesIO(b'')
-        p = pexpect.popen_spawn.PopenSpawn(' '.join([sys.executable, "-m", "deeppavlov", "interact", str(config_path)]),
-                                           timeout=None, logfile=logfile)
-        try:
-            for *query, expected_response in qr_list:  # works until the first failed query
-                for q in query:
-                    p.expect("::")
-                    p.sendline(q)
-
-                p.expect(">> ")
-                if expected_response is not None:
-                    actual_response = p.readline().decode().strip()
-                    assert expected_response == actual_response, \
-                        f"Error in interacting with {model_directory} ({config_path}): {query}"
-
-            p.expect("::")
-            p.sendline("quit")
-            p.readlines()
-            if p.wait() != 0:
-                raise RuntimeError('Error in quitting from deep.py: \n{}'.format(logfile.getvalue().decode()))
-        except pexpect.exceptions.EOF:
-            raise RuntimeError('Got unexpected EOF: \n{}'.format(logfile.getvalue().decode()))
+    def interact(config_path, qr_list=None, check_outputs=True):
```

Comment:
Don't we have to stop calling the method `interact` when migrating from *interact* to *infer* behavior ?